### PR TITLE
plan: remove useless aggregation function during "buildQuantifierPlan"

### PR DESCRIPTION
**Pull request by zz-jason**

### CODE REVIEW

**Comment by zz-jason** [comment]
@hanfei1991 @coocood PTAL

**Comment by zz-jason** [comment]
this blocks https://github.com/pingcap/tidb/pull/5490

File: plan/expression_rewriter.go [hunk 1]
```diff
@@ -354,60 +354,67 @@ func (er *expressionRewriter) handleCompareSubquery(v *ast.CompareSubqueryExpr)
...
 // buildQuantifierPlan adds extra condition for any / all subquery.
 func (er *expressionRewriter) buildQuantifierPlan(agg *LogicalAggregation, cond, rexpr expression.Expression, all bool) {
-	isNullFunc := expression.NewFunctionInternal(er.ctx, ast.IsNull, types.NewFieldType(mysql.TypeTiny), rexpr.Clone())
-	sumFunc := aggregation.NewAggFunction(ast.AggFuncSum, []expression.Expression{isNullFunc}, false)
-	countFuncNull := aggregation.NewAggFunction(ast.AggFuncCount, []expression.Expression{isNullFunc.Clone()}, false)
-	agg.AggFuncs = append(agg.AggFuncs, sumFunc, countFuncNull)
-	posID := agg.schema.Len()
-	aggColSum := &expression.Column{
+	funcIsNull := expression.NewFunctionInternal(er.ctx, ast.IsNull, types.NewFieldType(mysql.TypeTiny), rexpr.Clone())
+
```
Comment (zimulala) [inline plan/expression_rewriter.go]: Why add this blank?

**Comment by zimulala** [comment]
Rest LGTM

File: plan/expression_rewriter.go (hunk 1 above)
Comment (zz-jason) [reply plan/expression_rewriter.go]: for readability

**Pull request merged**